packages/contracts-bedrock/src/universal/OptimismMintableERC20.sol

```diff
@@ -120,4 +122,14 @@ contract OptimismMintableERC20 is IOptimismMintableERC20, ILegacyMintableERC20,
     function bridge() public view returns (address) {
         return BRIDGE;
     }
+
+    /// @dev Returns the number of decimals used to get its user representation.
+    /// For example, if `decimals` equals `2`, a balance of `505` tokens should
+    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
+    /// NOTE: This information is only used for _display_ purposes: it in
+    /// no way affects any of the arithmetic of the contract, including
+    /// {IERC20-balanceOf} and {IERC20-transfer}.
+    function decimals() public view virtual override returns (uint8) {
```

Comment:
Does this need to be virtual?

Reply:
nah it definitely does not since it's supplied via the constructor and not a constant as int he ERC20 base contract

Reply:
This needs test coverage where value is not 18